src/store.js

```diff
@@ -57,16 +63,12 @@ export const initialState = {
   },
   selectorReducer: {
```

Comment:
Is there a reason why we're keeping `selectorReducer` in the store?

Reply:
Entities still need to be moved out. That will be the next PR ...